### DISCUSSION TITLE
React v19 support

### DISCUSSION THREAD

**Discussion by Karmalakas** [ctrlplusb/easy-peasy#931] [General]
Funny enough, I was upgrading all dependencies last night for my app and `ncu` showed me there's a v18 of React (I was still using v16). It wasn't too much effort to upgrade to v18 and right after I did, I again checked with `ncu` and there it was - v19.0.0 😆 An hour or so later v19.0.1 appeared 😆 

So my question - is there some timeline maybe for easy-peasy to support React v19? 🙂

**Comment by jmyrland** [comment]
Yes, React versions > 18 was supported version [6.1.0 ](https://github.com/ctrlplusb/easy-peasy/releases/tag/v6.1.0)

**Reply by Karmalakas** [reply]
I literally just half an hour ago updated EP to 6.1, but left React on v18 😅 Will try to upgrade that too a bit later. Thanks 🙏
